rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/Experience.java

```diff
@@ -1,22 +1,21 @@
 package pl.oaza.warszawa.dor.rekolekcje.api.participants.domain;
 
+import static pl.oaza.warszawa.dor.rekolekcje.api.participants.domain.DateConverter.convertToUtc;
+
 import com.google.common.collect.Sets;
+import java.time.LocalDateTime;
+import java.util.Set;
+import java.util.stream.Collectors;
```

Comment:
\[Checkstyle\] WARNING: Import statement for 'java\.util\.stream\.Collectors' is in the wrong order\. Should be in the 'STANDARD\_JAVA\_PACKAGE' group, expecting group 'THIRD\_PARTY\_PACKAGE' on this line\.